### PR TITLE
Add iPad 6

### LINKED ISSUE #62: Add iPad 6

### PR DESCRIPTION
**Pull request by tbaranes**
Close #62

### CODE REVIEW

File: Source/iOS/Device.swift [hunk 1]
```diff
@@ -44,6 +44,7 @@ open class Device {
             case "iPad3,1", "iPad3,2", "iPad3,3":            return Version.iPad3
             case "iPad3,4", "iPad3,5", "iPad3,6":            return Version.iPad4
             case "iPad6,11", "iPad6,12":                     return Version.iPad5
+            case "iPad7,5", "iPad7,6":                       return Version.iPad5
```
Comment (SunLn) [inline Source/iOS/Device.swift]: Should it be `return Version.iPad6` here?

Comment (tbaranes) [reply Source/iOS/Device.swift]: wups

**Comment by tbaranes** [comment]
@Ekhoo up 🙏

**Comment by Ekhoo** [comment]
[Device v3.1.2](https://cocoapods.org/pods/Device) is available.

**Pull request closed without merging**